setup.py

```diff
@@ -95,14 +95,16 @@ def main():
                 "uvicorn[standard]==0.22.0",
                 "sentry-sdk[fastapi]==1.28.1",
                 "python-decouple==3.8",
+                "sphinx==6.2.1",
+                "sphinx_rtd_theme==1.2.2",
             ],
             "all": [
                 "colorama>=0.4.3",  # colorama
                 "fastapi[all]==0.99.1",  # app requirements
                 "uvicorn==0.22.0",
                 "sentry-sdk[fastapi]>=1.28.1",
                 "python-decouple==3.8",
-                "black==23.9.1",  # code formatting requirements
+                "black==23.12.1",  # code formatting requirements
```

Comment:
IMO, we should remove version for black and pylint, anyway it does not impact the application.

Reply:
I agree.